FCCee/compact/FCCee_o1_v02/LumiCal_o3_v02_02.xml

```diff
@@ -0,0 +1,143 @@
+<lccdd>
+    
+    <define>
+            <constant name="LumiCal_cell_size" value="1.81*mm"/>
+    </define>
+    
+    <readouts>
+        <readout name="LumiCalCollection">
+	    <segmentation type="PolarGridRPhi"
+			  grid_size_r="(LumiCal_outer_radius-LumiCal_inner_radius)/32"
+			  grid_size_phi="360/32*degree"
+			  offset_r="LumiCal_inner_radius + grid_size_r/2."
```

Comment:
`grid_size_r` in this line does not know what the value from two lines above

Reply:
I am also not sure about the half, if you compare to [this](https://github.com/iLCSoft/lcgeo/blob/master/CLIC/compact/CLIC_o3_v14/LumiCal_o1_v01_01.xml#L8)

Reply:
yes, that's the correct value:
 offset_r="LumiCal_inner_radius + (LumiCal_outer_radius-LumiCal_inner_radius)/64"

Reply:
@Voutsi so, I should only change a value of offset_r to the expression above, right?

Reply:
fixed